HW19/HW19_1.py

```diff
@@ -0,0 +1,48 @@
+class Bot:
+    def __init__(self, name):
+        self.name = name
+
+    def say_name(self):
+        print(self.name)
+
+    def send_message(self, message):
+        print(message)
+
+
+class TelegramBot(Bot):
+    def __init__(self, name):
+        super().__init__(name)
+        self.url = None
+        self.chat_id = None
+
+    def send_message(self, message):
+        if self.url is None:
+            url_str = "None"
+        else:
+            url_str = str(self.url)
+
+        if self.chat_id is None:
+            chat_id_str = "None"
+        else:
+            chat_id_str = str(self.chat_id)
```

Comment:
nit: this logic is not required because in f-string it will convert automatically